src/shared/src/persistence.ts

```diff
@@ -1,4 +1,4 @@
-export type StorageValue = string | number;
+export type StorageValue = string; // this is the ONLY type that SetStoredData accepts, it silently fails otherwise
```

Comment:
Should we not just remove the type ?

Reply:
I didn't want to do that just yet... although we probably will have to unless we can find a trick to allow us to make it work.

Reply:
> although we probably will have to unless we can find a trick to allow us to make it work.

What do you mean by that exactly ?

Reply:
Resolved on Discord. I have added a usePersistentNumberProperty hook for users that want to store numbers.